quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/VaultKeystorePropertyMappers.java

```diff
@@ -0,0 +1,32 @@
+package org.keycloak.quarkus.runtime.configuration.mappers;
+
+import org.keycloak.config.VaultOptions;
+
+import static org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.fromOption;
+
+final class VaultKeystorePropertyMappers {
+
+    private VaultKeystorePropertyMappers() {
+    }
+
+    public static PropertyMapper[] getVaultPropertyMappers() {
+        return new PropertyMapper[] {
+                fromOption(VaultOptions.VAULT)
+                        .paramLabel("provider")
+                        .build(),
+                fromOption(VaultOptions.VAULT_FILE)
+                        .to("kc.spi-vault-keystore-file")
+                        .paramLabel("file")
+                        .build(),
+                fromOption(VaultOptions.VAULT_PASS)
+                        .to("kc.spi-vault-keystore-pass")
+                        .paramLabel("pass")
+                        .build(),
+                fromOption(VaultOptions.VAULT_TYPE)
```

Comment:
I think we should use it to resolve the type depending on how FIPS is enabled too. See https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java#L146.

Reply:
Thanks for the review, @pedroigor. Is this how you meant it?
https://github.com/keycloak/keycloak/pull/19644/commits/2488d32d7692be5eb330d5256b36ce9e381dbf5d#diff-08975c153576d59b1d82e056efd2ba2cc67d82dd1e330c2fc78f4b91ab664d55R34-R56